src/common/Shape.js

```diff
@@ -17,6 +17,7 @@
     Shape.prototype.publish("height", 24, "number", "Height",null,{tags:['Private']});
     Shape.prototype.publish("colorStroke", null, "html-color", "Stroke Color", null, {tags:['Private']});
     Shape.prototype.publish("colorFill", null, "html-color", "Fill Color", null, {tags:['Private']});
+    Shape.prototype.publish("radius", null, "number", "Radius",null,{tags:['Private']});
```

Comment:
Once properties are published the "overriding" methods need to call to "new" corresponding base class methods